doc/nrf/ug_matter_device_certification.rst

```diff
@@ -33,32 +246,11 @@ The following table lists Matter over Thread certification requirements for when
 +-------------------------------+---------------------------+-----------------------------+----------------------------------------+
 | Thread Group Certification    | Production                | `Join Thread Group`_        | Implementer                            |
 +-------------------------------+---------------------------+-----------------------------+----------------------------------------+
-| Matter Certification          | Production                | `Join CSA`_                 | Adopter                                |
-+-------------------------------+---------------------------+-----------------------------+----------------------------------------+
 
 Bluetooth and Thread certifications can be inherited from Nordic Semiconductor (see the following section).
-Matter certification requires testing performed at Authorized Test Laboratory (ATL).
-
-To read more about Matter and dependent certification, get familiar with CSA's Certification Policy and Procedures document.
-
-.. _ug_matter_device_certification_reqs_security:
-
-Matter Attestation of Security
-==============================
-
-For a Matter end product to be certified, CSA's policies require an Attestation of Security that provides detailed information about the security level of the end product.
-The attestation document lists robustness security requirements based on the Matter specification.
-The product developer must indicate the level of compliance and briefly justify the choice.
-
-The attestation must be filled by the person responsible for end product certification who meets the following requirements:
-
-* The person's organization is a `member of the Matter community <Join CSA_>`_.
-* The person has an account on the `Connectivity Standards Alliance Certification Web Tool`_.
-
-Once both these requirements are met, the responsible person can download the `Matter Attestation of Security template`_, fill it in, and submit it in the certification web tool when applying for the certification.
 
-Matter certification inheritance
-********************************
+Matter dependent certification inheritance
+==========================================
```

Comment:
Line 260 and 261: 

* Bluetooth Qualified Design IDs (Bluetooth QDIDs) - **O**btained in accordance with `Bluetooth SIG's Qualification Process`_.
* Thread Certification IDs (Thread CIDs) - **O**btained in accordance with `Thread Group's certification information`_.